wallet/storage/src/internal/test.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+use common::chain::tokens::OutputValue;
+use common::chain::{Destination, OutPointSourceId, OutputPurpose, TxOutput};
+use common::primitives::{Amount, Id, H256};
+use crypto::key::{KeyKind, PrivateKey};
+use crypto::random::Rng;
+use rstest::rstest;
+use test_utils::random::{make_seedable_rng, Seed};
+
+impl Store<storage_sqlite::Sqlite> {
+    /// Create a default storage (mostly for testing, may want to remove this later)
+    pub fn new_in_memory() -> crate::Result<Self> {
+        Self::new(storage_sqlite::Sqlite::new_in_memory())
+    }
+}
+
+type TestStore = Store<storage_sqlite::Sqlite>;
+
+#[test]
+fn test_storage_get_default_version_in_tx() {
+    utils::concurrency::model(|| {
+        let store = TestStore::new_in_memory().unwrap();
+        let vtx = store.transaction_ro().unwrap().get_storage_version().unwrap();
+        let vst = store.get_storage_version().unwrap();
+        assert_eq!(vtx, 1, "Default storage version wrong");
+        assert_eq!(vtx, vst, "Transaction and non-transaction inconsistency");
+    })
+}
+
+#[cfg(not(loom))]
+#[rstest]
+#[trace]
+#[case(Seed::from_entropy())]
+fn utxo_test(#[case] seed: Seed) {
```

Comment:
Better than prefixing or suffixing test names with `test`, it's better to write something about the test. Like `read_write_utxo`, or `read_write_utxo_in_db_transaction`. This applies to any test.